docs/workflow/blocks/link_action_block.rst

```diff
@@ -0,0 +1,101 @@
+Link Action block
+===========
+The Link Action block provides a user-friendly button that links to other Flows. 
+Like a normal link, the user can choose how Flows are opened, according to their browser configuration.
+
+Existing Flows that use a combination of an :doc:`Actions block <actions>` and a :doc:`Launch Block <launch_block>` to open a Flow, can be replaced by the Link Action block to provide the user with more control.
+
+What problem does it solve?
+---------------------------
+Often when browsers present buttons to navigate to new screens, (such as when a Flow uses an Actions block with a Launch Block), the user is not given control over where the browser will open the new screen. Instead of allowing users to open the new screen in a new tab or window, the user may be forced to navigate away from their current screen, which unfortunately could cause context from the current tab to be lost!
+
+
+To provide the user with more control, the Link Action block can be used instead to produce buttons that allow opening a specified Flow in a new tab or window, or even the same browser context.
+
+For example, when a laptop user right-clicks a button generated by the Link Action block, their browser will typically present a context menu, then the user may choose to open the specified Flow in a new tab, allowing them to keep the existing Flow they have been using, as it is - without needing to navigate away from it.
+
+To perform the same behavior without using a right mouse button users typically hold the Cmd or Ctrl button and then click (Ctrl+click for Windows/Linux, and Cmd+click for macOS).
+
+Without relying on browser pop-ups, the Link Action block links to Flows in the browser's chosen context by using a browser anchor tag, as normally used for web page links, with the link styled as a button for a consistent user experience.
+
+By default, links to Flow's made with the Link Action block will open in the same window/tab context, but users with a mouse input device may right-click to open the link in a new tab or window, or on mobile devices with touchscreens, typically users would press and hold to open a Flow link produced by the Link Action block in a new tab.
+
+On supported browsers, users may notice the destination Flow is shown when hovering over the link, this allows the user to directly see where the button would navigate to.
+
+Default configuration
+--------------
+
+.. code-block:: json
+
+    {
+      "type": "link-action",
+      "label": "Open Workflow",
+      "adapterName": "adapterName",
+      "workflowId": "workflowId"
+    }
+
+Supported Properties
+--------------------
+
+- **label** (string): The text displayed on the button-like link. Defaults to "Link".
```

Comment:
The "other place" is Line 15 of this file. And I still see `"label": "Open Flow",`
@lukestanley Did you fix it somewhere else, or I still see an older version?

Reply:
When the block is firstly created, will the label configuration show `Link` or it will show `open Flow`? 
This is the default configuration to my understanding.